operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -1911,6 +1911,321 @@ gateway:
         hosts:
           - gateway.gem.example.com
 
+graphite:
+  # If true, enables graphite querier and graphite write proxy functionality.
+  # Read more in https://grafana.com/docs/metrics-enterprise/latest/graphite/
+  enabled: true
+
+graphite_querier:
+  replicas: 2
+
+  schemasConfiguration:
+    storageSchemas: |-
+      [default]
+      pattern = .*
+      intervals = 0:1s
+      retentions = 10s:8d,10min:1y
+    storageAggregations: |-
+      [default]
+      aggregationMethod = avg
+      pattern = .*
+      xFilesFactor = 0.1
+
+  service:
+    annotations: {}
+    labels: {}
+
+  resources:
+    requests:
+      cpu: 100m
+      memory: 128Mi
+
+  # Additional Grafana Enterprise Metrics container arguments, e.g. log level (debug, info, warn, error)
+  extraArgs: {}
+
+  # Pod Labels
+  podLabels: {}
+
+  # Pod Annotations
+  podAnnotations: {}
+
+  nodeSelector: {}
+  affinity:
+    podAntiAffinity:
+      preferredDuringSchedulingIgnoredDuringExecution:
+        - weight: 100
+          podAffinityTerm:
+            labelSelector:
+              matchExpressions:
+                - key: target
+                  operator: In
+                  values:
+                    - graphite-querier
+            topologyKey: 'kubernetes.io/hostname'
+
+  annotations: {}
+  persistence:
```

Comment:
The mimir image does otherwise it fails to start. The persistence.subPath key here is left just in case the user wants to change the volume path I guess?

Reply:
what does it fail with?

Reply:
With this https://github.com/grafana/mimir/pull/2711#discussion_r950084807

Reply:
replied in that comment